File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/body/user/ValidateCodeForChangePhoneNumberBody.java
package com.lattis.ellipse.data.network.model.body.user;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 3/15/17.
 */

public class ValidateCodeForChangePhoneNumberBody {

    @SerializedName("confirmation_code")
    private String confirmation_code;
    @SerializedName("phone_number")
    private String phoneNumber;

    public ValidateCodeForChangePhoneNumberBody(String confirmation_code, String phoneNumber) {
        this.confirmation_code = confirmation_code;
        this.phoneNumber=phoneNumber;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/body/user/ValidateCodeForChangePhoneNumberBody.kt
package com.lattis.data.entity.body.user

import com.google.gson.annotations.SerializedName

class ValidateCodeForChangePhoneNumberBody(
    @field:SerializedName("confirmation_code") private val confirmation_code: String,
    @field:SerializedName("phone_number") private val phoneNumber: String
)